boards/arm/frdm_kl46z/doc/index.rst

```diff
@@ -0,0 +1,225 @@
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/frdm_kl46z/frdm_kl46z_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The KL46Z SoC has five pairs of pinmux/gpio controllers, and all are currently enabled
+(PORTA/GPIOA, PORTB/GPIOB, PORTC/GPIOC, PORTD/GPIOD, and PORTE/GPIOE) for the FRDM-KL46Z board.
+
++-------+-------------+---------------------------+
+| Name  | Function    | Usage                     |
++=======+=============+===========================+
+| PTE22 | ADC         | ADC0 Light Sensor         |
++-------+-------------+---------------------------+
+| PTD5  | GPIO        | Red LED                   |
++-------+-------------+---------------------------+
+| PTE29 | GPIO        | Green LED                 |
++-------+-------------+---------------------------+
+| PTA0  | UART0_RX    | UART Console              |
++-------+-------------+---------------------------+
+| PTA1  | UART0_TX    | UART Console              |
++-------+-------------+---------------------------+
+| PTE25 | I2C0_SCL    | I2C                       |
++-------+-------------+---------------------------+
+| PTE24 | I2C0_SDA    | I2C                       |
++-------+-------------+---------------------------+
+| PTB16 | TSI0_CH9    | Capacitive Touch Slider   |
++-------+-------------+---------------------------+
+| PTB17 | TSI0_CH10   | Capacitive Touch Slider   |
++-------+-------------+---------------------------+
+
+
+System Clock
+============
+
+The KL46Z SoC is configured to use the 8 MHz external oscillator on the board
+with the on-chip FLL to generate a 48 MHz system clock.
+
+Serial Port
+===========
+
+The KL46Z UART0 is used for the console.
+
+USB
+===
+
+The KL46Z SoC has a USB OTG (USBOTG) controller that supports both
+device and host functions through its mini USB connector (USB KL46Z).
+Only USB device function is supported in Zephyr at the moment.
+
+LCD Display
+===========
+
+FRDM-KL46Z is using a 4 digit display (LUMEX LCD-S401M16KR) 4x8 segments. Currently non supported by this port.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`.
+
+Early versions of this board have an outdated version of the OpenSDA bootloader
+and require an update. Please see the `DAPLink Bootloader Update`_ page for
```

Comment:
You'll need to define the target for this link (as you have for other targets at the bottom of the doc)